src/index.js

```diff
@@ -7,25 +7,7 @@ const CacheService = require('./cache/CacheService')
 const PropertiesService = require('./properties/PropertiesService')
 const UrlFetchApp = require('./url-fetch/UrlFetchApp')
 const Session = require('./base/classes/Session')
-
```

Comment:
I think it would be hard to manage this with the number of services grows. 
And probably developers won't use import of exact class often. Those who need exact class import can do this with `import ... from 'gas-mock-globals/src/card/classes/Card'`
Also I've added note to README: `use exact class without global namespace pollution`